lib/util/baseValidationError.ts

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+import { Severity } from "./severity"
+import { ValidationResultSource } from "./validationResultSource"
+
+export interface BaseValidationError {
```

Comment:
baseValidationError vs CommonError in error.ts ? is any of them redundant?  Otherwise can we add a comment for what is each for ?

Reply:
Eh, I wasn't able to use one type for now. We may revise this latter.